documentation/config_file_puppetserver.md

```diff
@@ -90,6 +99,12 @@ http-client: {
 profiler: {
     enabled: true
 }
-~~~
+
+# Settings related to static catalogs.
+versioned-code: {
+    code-id-command: /opt/puppetlabs/server/apps/puppetserver/code-id-command_script.sh
+    code-content-command: /opt/puppetlabs/server/apps/puppetserver/code-content-command_script.sh
+}
+```
```

Comment:
Would it be worth noting that these are hypotethical file paths for these user-created scripts?  I worry that, just glancing at this example config, a user might get the idea that those scripts are included with Puppet Server and should be that location.